<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>pythonUtilsConfigWidget</class>
    <widget class="QWidget" name="pythonConfigWidget">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                        <widget class="QGroupBox" name="groupBox">
                            <property name="title">
                                <string>PEP-8 config</string>
                            </property>
                            <layout class="QHBoxLayout" name="horizontalLayout">
                                <item>
                                    <widget class="QLabel" name="label">
                                        <property name="text">
                                            <string>Check PEP8 when save</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QCheckBox" name="checkPEP8WhenSave">
                                </widget>
                                </item>
                                <item>
                                    <widget class="QLabel" name="label">
                                        <property name="text">
                                            <string>Ignore PEP8 errors</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="ignorePEP8Errors">
                                </widget>
                                </item>
                            </layout>
                        </widget>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>Check PyFlakes when save</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QCheckBox" name="checkPyFlakesWhenSave">
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>Check Syntax Errors when save</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QCheckBox" name="checkParseCode">
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>IPython pdb snippet</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="ipdbSnippet">
                                    <property name="value">
                                        <string>import ipdb; ipdb.set_trace()</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <spacer name="verticalSpacer">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>20</width>
                                    <height>40</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
